calculator.py

```diff
@@ -1,11 +1,26 @@
 def addition(number_1, number_2):
+    print("sum of",number_1," and ",number_2," is ",(number_1 + number_2))
```

Comment:
perform addition first then print the result.
In future if you have to add an array what will you do.